backend/tests/conftest.py

```diff
@@ -8,3 +10,10 @@ def client() -> TestClient:
 
     app = create_app()
     return TestClient(app)
+
+
+@pytest.fixture(autouse=True)
```

Comment:
How about you move those fixtures to a subdirectory (e.g. backend/tests/api/) so they are not included in other tests that don't need them?

Reply:
Maybe. The fixture I've added in this PR is imo essential and it should stay here for improved test isolation.

Reply:
So maybe at least move the client fixture?